MonacoEditorComponent/MonacoEditorComponent.csproj

```diff
@@ -11,8 +11,8 @@
     <AssemblyName>MonacoEditorComponent</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17134.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.14393.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17763.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.16299.0</TargetPlatformMinVersion>
```

Comment:
I had originally bumped these up before, but then had trouble building my dependent project in Azure DevOps, so I'm a bit weary to update the min to 16299 unless needed.  I also haven't been able to update my home machine to 17763 yet due to issues from the manufacturer :(.  

Were there other changes you were making that required the update?

Reply:
17763 was because that is the SDK I’m using on my machine, and I thought its the recommended Windows version now.

16299 because that is the minimum version when you can start using .net standard 2.0 and I believe older builds are not supported anymore. I believe e.g. Windows Community Toolkit moved to that for the same reason (see https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/1954).

Maybe there is a solution for your issue in Azure DevOps?